openfisca_uk/variables/finance/benefit/family/LHA.py

```diff
@@ -32,9 +32,9 @@ def formula(benunit, period, parameters):
             not_(male) * (benunit.members("age", period) < 16)
         )
         return (
-            (children_under_10 > 0)
-            + (under_16_male > 0)
-            + (under_16_female > 0)
+            (children_under_10 > 0).astype(int)
```

Comment:
for the future, let's make `10` and `16` policy params

Reply:
Agreed, leaving in for now